runtime/doc/ui.txt

```diff
@@ -170,7 +174,9 @@ UIs).
 Most of these events take a `grid` index as first parameter.  Grid 1 is the
 global grid used by default for the entire editor screen state. Grids other
 than that will be defined by future extensions. Just activating the `ext_newgrid`
-option by itself will never cause any additional grids to be created.
+option by itself will never cause any additional grids to be created. To
+enable per-window grids, `ext_multigrid` option must be set (see
```

Comment:
To enable per-window grids, set the `ext_multigrid` option. (see |ui-multigrid|)